tests/templates/kuttl/resources/10-assert.yaml.j2

```diff
@@ -47,14 +41,11 @@ metadata:
   name: resources-crd-exec-1
 spec:
   containers:
-{% if lookup('env', 'VECTOR_AGGREGATOR') %}
-    - name: vector
-{% endif %}
     - name: spark
       resources:
         limits:
           cpu: "1"
           memory: 1Gi
         requests:
-          cpu: 250m
+          cpu: "1"
```

Comment:
Could we please add a comment like
`This resources are set from spark driver according to the Spark config`